### ISSUE TITLE
fresh install, but unable to start

### ISSUE THREAD

**Issue by jackielii** [fannheyward/coc-pyright#274]
`:messages` says:

```
[coc.nvim] Server pyright failed to start: TypeError: Cannot read property 'Deprecated' of undefined
```

**Comment by jackielii** [comment]
hmm, just switch to master of coc.nvim and the error seems to have gone

**Comment by samyak-jain** [comment]
@jackielii Had the same issue. Your suggestion seems to work

**Comment by fannheyward** [comment]
Fixed in v1.1.104, thank you for your report.
